concourse/rbac/dev-team.yaml

```diff
@@ -0,0 +1,10 @@
+roles:
```

Comment:
we might not want to name this directory rbac. concourse *also* supports k8s rbac, and if we decide to adopt it, it may seem confusing.

Reply:
Agree. There are much thing need to discuss before implement.